docs/how-to/llm-fine-tuning-optimization/llm-inference-frameworks.rst

```diff
@@ -52,7 +43,7 @@ Installing vLLM
    .. tab-item:: vLLM on a single-accelerator system
       :sync: single
 
-      3. To use vLLM as an API server to serve reference requests, first start a container using the :ref:`vllm-rocm
+      2. To use vLLM as an API server to serve reference requests, first start a container using the :ref:`vllm-rocm
```

Comment:
vllm-rocm -> vllm

Reply:
> vllm-rocm -> vllm

vllm-rocm is the docker image name built from previous step.